numalogic/registry/mlflow_registry.py

```diff
@@ -135,15 +135,15 @@ def load(
         self,
         skeys: Sequence[str],
         dkeys: Sequence[str],
-        latest: bool = True,
+        production: bool = True,
         version: str = None,
     ) -> Optional[ArtifactData]:
         model_key = self.construct_key(skeys, dkeys)
         try:
-            if (latest and version) or (not latest and not version):
+            if (production and version) or (not production and not version):
                 raise ValueError("Either One of 'latest' or 'version' needed in load method call")
 
-            elif latest:
+            elif production:
```

Comment:
line 148: checks if there is version_info
line 152: skips checking version_info

should we apply the same check in both cases?